### PR TITLE
Feature/update stage env

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Updating stage value

### CODE REVIEW

File: .env.stage [hunk 1]
```diff
@@ -0,0 +1,27 @@
+PORT=3000
+HTTPS=true
+
+REACT_APP_BLOCKNATIVE_DAPP_ID=b2f0b5aa-a953-4e7b-b92a-e6b1b40a10c6
```
Comment (AndreiNavoichyk) [inline .env.stage]: We don't need this key

File: .env.stage [hunk 2]
```diff
@@ -0,0 +1,27 @@
+PORT=3000
+HTTPS=true
+
+REACT_APP_BLOCKNATIVE_DAPP_ID=b2f0b5aa-a953-4e7b-b92a-e6b1b40a10c6
+REACT_APP_SUPPORT_SERVICE=
+REACT_APP_SUPPORT_URL=
+
+## These are only required if using Sentry for error logging
+REACT_APP_SENTRY_DSN_URL=
+REACT_APP_SENTRY_ENV=development
```
Comment (AndreiNavoichyk) [inline .env.stage]: Update with `staging`

**Review by AndreiNavoichyk — APPROVED** [review]

File: .env.stage (hunk 2 above)
Comment (karthikeyan-cere) [reply .env.stage]: Updated

File: .env.stage (hunk 1 above)
Comment (karthikeyan-cere) [reply .env.stage]: Removed

**Pull request merged**